frontend/src/components/DataTable.vue

```diff
@@ -1,29 +1,44 @@
 <template>
-  <div class="flex justify-end gap-2 mb-2">
-    <SearchField
-      v-if="config.search"
-      v-model="search"
-    />
-    <label
-      v-for="filter in filters"
-      :key="filter"
+  <div class="grid grid-cols-2 gap-2">
```

Comment:
I would actually prefer flex and justify-between here, since the left and the right side might have a different amount of items. With grid-cols-2, they will only be allowed to use half the width.

Reply:
Sure, can you share what you have in mind ? I'm not familiar with flex.



( Btw, everything I add that is UI related can be thrown out in the garbage of you want 😂, I hate CSS 😂)

Reply:
It's actually not that bad 😉 have a look at tailwind flex utilities. It pretty much just "flex justify-between" on the parent and no class on the children